client/common/hydrate.ts

```diff
@@ -1,8 +1,10 @@
 'use strict';
 
+import { I18n } from './i18n';
 import { determFormat } from "./formatting";
 import { format } from "./formatting";
 
+const currI18n = new I18n;
```

Comment:
i think you want:

```
import I18ns from './i18n';
```

and then to translate, you can go:

```
I18ns.__(noticePB.notice.content, { prefix: '<strong>', postfix: '</strong>' })
```

but @dropmann, we can probably simplify the i18ns interface? i.e.

```
import { __ } from './i18n';

__(...)
```

??

Reply:
Resolved.